app/packages/looker/src/elements/common/bubbles.ts

```diff
@@ -54,14 +54,14 @@ export const getBubbles = (
     }
 
     if (field.embeddedDocType === withPath(LABELS_PATH, CLASSIFICATIONS)) {
-      out.values = out.values.flatMap(
+      out.values = unwind(field.dbField, out.values).flatMap(
```

Comment:
What does out.values look like? Flat map should unnest arrays so it's weird that you would need to unwind first

Reply:
out.values is a list or single value of potentially renderable field data. flatMap flattens the nested `classifications`